src/classes/Inventory.ts

```diff
@@ -195,6 +196,44 @@ export default class Inventory {
         return nonTradable.concat(tradable).slice(0);
     }
 
+    findByPartialSku(partialSku: string, tradableOnly = true, elevatedStrange = false): string[] {
+        const matchingSkus: string[] = [];
+
+        if (elevatedStrange) {
+            partialSku = partialSku.replace(';strange', '');
+
+            for (const sku in this.tradable) {
+                if (sku.startsWith(partialSku) && sku.includes(';strange')) {
+                    matchingSkus.push(...this.tradable[sku].map(item => item?.id));
+                }
+            }
+
+            if (!tradableOnly) {
+                for (const sku in this.nonTradable) {
+                    if (sku.startsWith(partialSku) && sku.includes(';strange')) {
+                        matchingSkus.push(...this.nonTradable[sku].map(item => item?.id));
+                    }
+                }
+            }
```

Comment:
I think this is not necessary, as it's impossible to add items that are not tradeable.

Reply:
Removed check